Add "No Oracle DB hostname reported" to known issues

keisku opened the pull request:
<!-- *Note: Please remember to review the Datadog Documentation [Contribution Guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) if you have not yet done so.* -->

### What does this PR do? What is the motivation?
<!-- A brief description of the change being made with this pull request. What is your motivation for the PR? -->

https://github.com/DataDog/datadog-agent/pull/34866

### Merge instructions

<!-- 
If you're waiting for a release or there are other considerations that you want us to be aware of, list them here. 
If the PR is ready to be merged once it receives the required reviews, check the box below after you've created the PR.
-->

Merge readiness:
- [ ] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### Additional notes
<!-- Anything else we should know when reviewing?-->

<!-- Previewing the PR: Assuming you are a Datadog employee and named your branch `<yourname>/<description>`, a preview build will run and links to the preview output will be auto-generated and posted in the PR comments. The links will 404 until the preview build is finished running. -->

nenadnoveljic commented:
Can we be more specific that this happens with Autonomous Database?

keisku commented:
@nenadnoveljic What do you think about the updated sentence?

nenadnoveljic commented:
> @nenadnoveljic What do you think about the updated sentence?

I suggest removing "at least".

buraizu commented on content/en/database_monitoring/setup_oracle/troubleshooting.md:
```diff
@@ -36,6 +36,17 @@ Ensure that the [recommended Agent version][2] for your hosting type is installe
+
+### No Oracle DB hostname reported
+
+Datadog Agent detects the Oracle DB hostname by running SQL against [V$INSTANCE][4].
```
```suggestion
The Datadog Agent detects Oracle DB hostnames by running SQL commands against [V$INSTANCE][4].
```

nenadnoveljic replied:
Should `hostnames` and `commands` be singular as the Agent runs only one query which returns only one hostname?

buraizu replied:
Thanks for that question, and apologies for the delay following up here. In this case, no, they should not be singular, as what's being referenced is a general behavior rather than a single occurrence of a behavior. 

If you feel that it's important to emphasize that the Agent runs only one query, which returns only one hostname, then an alternative phrasing would be:
The Datadog Agent detects **the** Oracle DB hostname by running **a** SQL command against [V$INSTANCE]

buraizu commented on content/en/database_monitoring/setup_oracle/troubleshooting.md:
```diff
@@ -36,6 +36,17 @@ Ensure that the [recommended Agent version][2] for your hosting type is installe
+### No Oracle DB hostname reported
+
+Datadog Agent detects the Oracle DB hostname by running SQL against [V$INSTANCE][4].
+When Oracle DB returns `null` for the `HOST_NAME` column, the Datadog Agent reports the Oracle DB host name as empty. We have confirmed this behavior with Oracle Autonomous Database.
```
```suggestion
When an Oracle DB returns `null` for the `HOST_NAME` column, the Datadog Agent reports the Oracle DB host name as empty. This behavior has been confirmed with Oracle Autonomous Database as well.
```

nenadnoveljic replied:
Shall we leave `as well` out? We have confirmed the behaviour *only* with Oracle Autonomous Database.

buraizu commented on content/en/database_monitoring/setup_oracle/troubleshooting.md:
```diff
@@ -36,6 +36,17 @@ Ensure that the [recommended Agent version][2] for your hosting type is installe
+
+Datadog Agent detects the Oracle DB hostname by running SQL against [V$INSTANCE][4].
+When Oracle DB returns `null` for the `HOST_NAME` column, the Datadog Agent reports the Oracle DB host name as empty. We have confirmed this behavior with Oracle Autonomous Database.
+In this case, we recommend setting the `reported_hostname` in the `conf.yaml`.
```
```suggestion
In this case, Datadog recommends setting the `reported_hostname` in the `conf.yaml` file.
```

buraizu commented on content/en/database_monitoring/setup_oracle/troubleshooting.md:
```diff
@@ -36,6 +36,17 @@ Ensure that the [recommended Agent version][2] for your hosting type is installe
+Datadog Agent detects the Oracle DB hostname by running SQL against [V$INSTANCE][4].
+When Oracle DB returns `null` for the `HOST_NAME` column, the Datadog Agent reports the Oracle DB host name as empty. We have confirmed this behavior with Oracle Autonomous Database.
+In this case, we recommend setting the `reported_hostname` in the `conf.yaml`.
+[Since v7.65.0 and above][5], the Datadog Agent will output logs if this condition is met.
```
```suggestion
[Since v7.65.0 and later][5], the Datadog Agent outputs logs if this condition is met.
```

buraizu commented on content/en/database_monitoring/setup_oracle/troubleshooting.md:
```diff
@@ -36,6 +36,17 @@ Ensure that the [recommended Agent version][2] for your hosting type is installe
 [2]: /database_monitoring/setup_oracle#recommended-agent-version
 [3]: /database_monitoring/setup_oracle#setup
+[4]: https://docs.oracle.com/en/database/oracle/oracle-database/23/refrn/V-INSTANCE.html
+[5]: https://github.com/DataDog/datadog-agent/pull/34866
```
```suggestion
[5]: https://github.com/DataDog/datadog-agent/releases/tag/7.65.0
```

buraizu reviewed (changes requested):
Thanks for the PR, requested some updates per the [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md). It looks like this is waiting on the release of Agent version `7.65.0`. Is there an expected timetable for that?

keisku commented:
> It looks like this is waiting on the release of Agent version 7.65.0. Is there an expected timetable for that?

I removed the reference of v7.65.0. Other than that, I updated the sentence as your suggestion. Thank you.

buraizu commented on content/en/database_monitoring/setup_oracle/troubleshooting.md:
```diff
@@ -36,6 +36,15 @@ Ensure that the [recommended Agent version][2] for your hosting type is installe
+
+### No Oracle DB hostname reported
+
+The Datadog Agent detects Oracle DB hostname by running SQL command against [V$INSTANCE][4].
```
```suggestion
The Datadog Agent detects Oracle DB hostnames by running SQL commands against [V$INSTANCE][4].
```

buraizu replied:
Thanks for flagging that, followed up on the comment.

buraizu commented on content/en/database_monitoring/setup_oracle/troubleshooting.md:
```diff
@@ -36,6 +36,15 @@ Ensure that the [recommended Agent version][2] for your hosting type is installe
+### No Oracle DB hostname reported
+
+The Datadog Agent detects Oracle DB hostname by running SQL command against [V$INSTANCE][4].
+When an Oracle DB returns `null` for the `HOST_NAME` column, the Datadog Agent reports the Oracle DB host name as empty. This behavior has been confirmed with Oracle Autonomous Database.
```
```suggestion
When an Oracle DB returns `null` for the `HOST_NAME` column, the Datadog Agent reports the Oracle DB hostname as empty. This behavior has been confirmed with Oracle Autonomous Database.
```

buraizu reviewed (changes requested):
Minor updates to the new content

buraizu commented:
/merge
